metaci/db/management/commands/populate_db.py

```diff
@@ -22,19 +31,36 @@ def handle(self, *args, **options):
             Please clear the database before continuing!
             One way of clearing the database is:
 
-            python manage.py reset_db -c && python manage.py migrate
+            python manage.py reset_db -c && python manage.py migrate && python manage.py createsuperuser --username clark --email kent@krypton.com
 
             You can add a --noinput to that if you're sure you know what you're doing.
             """
             )
 
     def check_non_destructive(self):
-        return User.objects.count() < 2 and TestResult.objects.count() < 50
+        return User.objects.count() < 3 and TestResult.objects.count() < 10
 
     def create_objs(self):
-        user1 = fact.UserFactory()
-        user2 = fact.UserFactory()
-        superuser = fact.StaffSuperuserFactory()
-        TestMethod1 = fact.TestMethodFactory()
-        TestMethod2 = fact.TestMethodFactory()
-        (user1, user2, superuser, TestMethod1, TestMethod2)  # quiet linter
+        with transaction.atomic():
+            factory.random.reseed_random("TOtaLLY RaNdOM")
+            fact.UserFactory()
+            fact.UserFactory()
+            PublicPlanRepository = fact.PlanRepositoryFactory()
+            PrivatePlanRepository = fact.PlanRepositoryFactory()
+            assign_perm(
+                "plan.view_builds",
+                Group.objects.get(name="Public"),
+                PublicPlanRepository,
+            )
+
+            fact.TestResultFactory(build_flow__build__planrepo=PublicPlanRepository)
+            fact.TestResultFactory(build_flow__build__planrepo=PrivatePlanRepository)
+            fact.TestResultFactory(build_flow__build__planrepo=PublicPlanRepository)
+            fact.TestResultFactory(build_flow__build__planrepo=PrivatePlanRepository)
+
+            self.make_consistent()
```

Comment:
I think you could use a `post_generation`-decorated method on TestResultFactory to increment the build_flow's `tests_total`. But it would involve more writes. This is fine.

Reply:
Makes sense. Honestly, the only use I know of this db column is in some of my code that makes sure that it is greater than 0 to avoid looking at irrelevant build_flows. So it is probably not worth slowing down the test suite for. I probably should have just left it hard-coded to "5" as I did for the last month. ;)